app/templates/resources/read_resource.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <h1>Resource</h1>
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td><a target="_blank" href="{{ resource.website }}">{{ resource.website }}</a></td></tr>
+                <tr><td>Address</td><td>{{ resource.address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ resource.user.full_name() }}</td></tr>
+            </table>
+            <a href="{{ url_for('resources.create_review', resource_id=resource.id) }}">
+                <button class="ui button">Write a Review</button>
+            </a>
+        </div>
+
+        {% block form %}
+        {% endblock %}
+
+        {% for review in reviews %}
```

Comment:
having the for loop outside means that each review will have the `<h1>Reviews</h1>` above it. Can we move this for loop inside the nesting just under the `<h1>`.